scripts/ipet-evaluate

```diff
@@ -127,6 +127,7 @@ argparser.add_argument('-i', '--index', action = "append", default = None, help
 argparser.add_argument('--indexsplit', default = None, help = "position to split index into row and column levels, negative to count from the end.")
 argparser.add_argument('--quiet', action = "store_true", default = False, help = "Supress all output (may be useful for profiling)")
 argparser.add_argument('--displaygroup', default = None, help = "Name of the group for which the long display should be printed. Only available for long output mode")
+argparser.add_argument('--suppressions', default = None, help = "List of regular expressions of columns to have their output suppressed")
```

Comment:
I would appreciate if you could document this better in the epilog of "ipet-evaluate.py". There is already a section that documents how to modify the output.

Reply:
Added this to the epilog:

```
    Certain columns can be suppressed by using the --suppressions tag with a list of regular expressions:
        ipet-evaluate -t sometestrun.trn -e evaluation.xml --suppressions "<someRegex1> <someRegex2> ... <someRegexN>"
    For example, to suppress the column Time_shmean(1.0)p and all columns ending with Q in the aggregation table:
        ipet-evaluate -t sometestrun.trn -e evaluation.xml --suppressions "Time_shmean\(1\.0\)p .*Q$"
    Note that special characters in column names need to be escaped for the matching to work.
```